front/src/modules/auth/states/prefillLoginWithSeed.ts

```diff
@@ -0,0 +1,6 @@
+import { atom } from 'recoil';
+
+export const prefillLoginWithSeed = atom<boolean>({
+  key: 'prefillLoginWithSeed',
+  default: true,
```

Comment:
probably default behavior should be false for seeds, we usually want to fall back on the production behavior (if e.g. the clientConfig happens to fail in prod because user has bad network, then we don't want strange a strange behavior like this login happening)